etc/grafana/dashboards/overview.json

```diff
@@ -4769,6 +4769,99 @@
       ],
       "title": "File Descriptors",
       "type": "timeseries"
+    },
+    {
```

Comment:
This should also probably be placed with the rest of the peer related metrics (for example after peer disconnect reasons)